guides/common/modules/con_overview-of-installation.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="overview-of-the-installation-process"]
+= Overview of the installation process
+
+The {Project} installation process can be broken down into three main phases:
+* Planning
+* Server preparation
+* Installation and configuration
+
+.Planning
+Make sure that your infrastructure meets the prerequisites for installation:
+
+* Ensure your hardware and software meet the minimum requirements for CPU, RAM, and storage space.
+* Plan your network setup, including DNS resolution and port/firewall configurations.
+* For large deployments (e.g., more than 5,000 hosts), select a predefined performance profile to optimize your system.
+
+.Server preparation
+Configure the operating system and the repositoties for installating {Project}:
+
+* Install {EL} 9 on a freshly installed system.
```

Comment:
If we're rendering the overview for all builds, this needs to be changed for `foreman-deb` to `Debian 12 or Ubuntu 22.04` or something.